homework/src/index.js

```diff
@@ -30,17 +32,109 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+  async function main(url) {
+    const root = document.getElementById('root');
+    const header = createAndAppend('header', root);
+    createAndAppend('p', header, {html: 'HYF Repositories', class: 'header-lable'});
+    const select = createAndAppend('select', header, { id: 'selectMenu' });
+    // use promise object
+    try {
+      const data = await fetchJSON(url);
```

Comment:
Data is a very nice name, but it means on something anonymous, you have to be somehow creative with your names like what does it contains or something, maybe it's boring that is everyone else using it `repositories` but it's a meaningful one, or as I told be creative :heart:.

So What is data?